cannot find package "golang.org/x/tools/go/packages"

https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1530/pull-ci-azure-master-upgrade-v3.2/43/build-log.txt

/kind test-flake

```
bash-4.2$     make create                                                                                                                                                                                                                                                    
+ make create                                                                                                                                                                                                                                                                
PLUGIN_VERSION is v3.2
TAG set to v3.2
timeout 1h ./hack/create.sh mjudeikis
+ [[ 1 -ne 1 ]]
+ export RESOURCEGROUP=mjudeikis
+ RESOURCEGROUP=mjudeikis
+ rm -rf _data
+ mkdir -p _data/_out
+ set -x
+ [[ -n '' ]]
+ go generate ./...
bindata.go
bindata.go
../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/arm/arm.go:7: running "go": exit status 1
../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/cluster/hash.go:4: running "go": exit status 1
../../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/cluster/kubeclient/types.go:4: running "go": exit status 1
../../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/cluster/scaler/scaler.go:4: running "go": exit status 1
../../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/cluster/updateblob/blobservice.go:4: running "go": exit status 1
../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
        /tmp/tmp.04nftemodd/src/golang.org/x/tools/go/packages (from $GOPATH)
pkg/config/types.go:4: running "go": exit status 1
../../../../../golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
        /usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)

bash-4.2$ make create                                                                                                                                                                                                                                                        
+ make create
PLUGIN_VERSION is v3.2
TAG set to v3.2
timeout 1h ./hack/create.sh mjudeikis
+ [[ 1 -ne 1 ]]
+ export RESOURCEGROUP=mjudeikis
+ RESOURCEGROUP=mjudeikis
+ rm -rf _data
+ mkdir -p _data/_out
+ set -x
+ [[ -n '' ]]
+ go generate ./...
bindata.go
bindata.go
bindata.go
+ [[ -n '' ]]
+ trap 'return_id=$?; set +ex; kill $(lsof -t -i :8080); wait $(lsof -t -i :8080); exit $return_id' EXIT
+ go run cmd/fakerp/main.go
+ go run cmd/createorupdate/createorupdate.go
FATA[2019-04-26T11:33:51Z] required key AZURE_SUBSCRIPTION_ID missing value
exit status 1
+ return_id=1
+ set +ex
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
INFO[2019-04-26T11:33:52Z]/tmp/tmp.04nftemodd/src/github.com/openshift/openshift-azure/cmd/fakerp/main.go:20 main.main() starting the fake resource provider                                                                                                                 
FATA[2019-04-26T11:33:52Z]/tmp/tmp.04nftemodd/src/github.com/openshift/openshift-azure/pkg/fakerp/server.go:60 github.com/openshift/openshift-azure/pkg/fakerp.NewServer() open secrets/logging-int.cert: permission denied                                                  
exit status 1
make: *** [create] Error 1
```

--- a/hack/tests/e2e-scaleupdown-previous.sh
+++ b/hack/tests/e2e-scaleupdown-previous.sh
@@ -40,6 +40,10 @@ ln -sf "$PWD/secrets" "$T/src/github.com/openshift/openshift-azure"
     export AZURE_WORKER_CLIENT_ID=$AZURE_LEGACY_WORKER_CLIENT_ID
     export AZURE_WORKER_CLIENT_SECRET=$AZURE_LEGACY_WORKER_CLIENT_SECRET
     cd "$T/src/github.com/openshift/openshift-azure"
+    # TODO: remove after v3.2 goes away
+    if [[ "$1" == "v3.2" ]]; then
+        GOPATH="$T" go get github.com/golang/mock/mockgen
+    fi
     GOPATH="$T" make create
 )
 

--- a/hack/tests/e2e-upgrade.sh
+++ b/hack/tests/e2e-upgrade.sh
@@ -40,6 +40,10 @@ ln -sf "$PWD/secrets" "$T/src/github.com/openshift/openshift-azure"
     export AZURE_WORKER_CLIENT_ID=$AZURE_LEGACY_WORKER_CLIENT_ID
     export AZURE_WORKER_CLIENT_SECRET=$AZURE_LEGACY_WORKER_CLIENT_SECRET
     cd "$T/src/github.com/openshift/openshift-azure"
+    # TODO: remove after v3.2 goes away
+    if [[ "$1" == "v3.2" ]]; then
+        GOPATH="$T" go get github.com/golang/mock/mockgen
+    fi
     GOPATH="$T" make create
 )
 

